packages/icons/src/library/crop.js

```diff
@@ -5,7 +5,8 @@ import { SVG, Path } from '@wordpress/primitives';
 
 const crop = (
 	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-		<Path d="M16.5 7.8v7H18v-7c0-1-.8-1.8-1.8-1.8h-7v1.5h7c.2 0 .3.1.3.3zm-8.7 8.7c-.1 0-.2-.1-.2-.2V2H6v4H2v1.5h4v8.8c0 1 .8 1.8 1.8 1.8h8.8v4H18v-4h4v-1.5H7.8z" />
+		<Path d="M18 21v-3h3v-1.5H7.75a.25.25 0 0 1-.25-.25V3H6v3H3v1.5h3v8.75c0 .966.784 1.75 1.75 1.75h8.75v3H18Z" />
```

Comment:
Same open question around multiple paths as above.

Reply:
Thanks! Added to https://github.com/WordPress/gutenberg/pull/56532/commits/2ca146a78218e7fab43e9fb8014d0f669c83285c.